lib/node_modules/@stdlib/constants/float32/sqrt-pi/lib/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Square root of single-precision floating-point mathematical constant `π`.
```

Comment:
@gururaj1512 Instead of this, let's use:
```
Square root of the mathematical constant `π` as a single-precision floating-point number.
```
here and everywhere else in this package. This would be somewhat similar to what we have in https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/constants/float32/ln-two/lib/index.js#L22.